presto-orc/src/main/java/com/facebook/presto/orc/reader/DecimalStreamReader.java

```diff
@@ -152,6 +156,8 @@ private void assureVectorSize()
         if (nullVector.length < requiredVectorLength) {
             nullVector = new boolean[requiredVectorLength];
             scaleVector = new long[requiredVectorLength];
+
+            systemMemoryContext.setBytes(requiredVectorLength * 9);
```

Comment:
Instead of `9`, please use `SIZE_OF_LONG + SIZE_OF_BYTE`. These constants are defined in `io.airlift.slice.SizeOf`.